lips/lip-5.md

```diff
@@ -0,0 +1,158 @@
+
+A common pattern to a cross-wallet transaction would be:
+
+1) The recipient generates a *intent identifier* consisting of their account identifier, and payment metadata
+2) The recipient shares this information with a would-be sender over some communication channel 
+3) The sender's wallet deserializes this information and populates a transaction for the sending user to authorize
+4) The sending user authorizes the request and their wallet submits the transaction to the Libra blockchain
+5) The recipient wallet witnesses the transaction on-chain and confirms the funds have been received
+
+Examples of cross-wallet payment scenarios include:
+
+* A user may want to share a compact identifier over traditional communication channels (eg. SMS) and receive payment from a user of a different wallet
+    * The recipient user may want to specify a currency preference
+    * The recipient user may want to specify a concrete amount they'd like to receive
+* A merchant may want to present a predefined payment request via QR code so that a customer would only need to authorize the funds
+
+
+# Specification
+
+## Account identifiers
+For communicating account identity, we propose using a compact, versioned and case-insensitive identifier. To meet this criteria, we selected the Bech32 encoding implementation used in Bitcoin Segwit (BIP 0173) excluding the Segwit byte known-length restrictions.
+
+### Desired attributes
+- Consistent - Users can build a muscle memory for identifying and using these account addresses
+- Atomic - The string identifier feels like a single unit. Users shouldn’t try to separate or truncate the string
+- Versioned - The format contains human readable information about how to interpret the payload, preventing subtle errors and reducing risk of linkability for human readable addresses
+- Error detecting - Bech32 checksums help clients validate input and reduce risk of bad transactions due to mistypings and truncations
+
+### Constraints for subaddresses
+- Fixed length at 8 bytes (we don’t accept other sizes, strictly 8 bytes).
+- Unique to their custodian/on-chain account: addresses must be non-ambiguous (ie. Recipient VASPs should not issue the same address to multiple users).
+- By convention, 8 zero bytes (`0x0000000000000000`) is reserved to denote the root (VASP owned) account.
+
+### Format
+The Libra Account Identifier consists of 
+* A prefix (also known as hrp (human readable part) which identifies the network version this address is intended for
+  * “lbr” for Mainnet addresses
+  * “tlb” for Testnet addresses
+* A Bech32 delimiter
+  * The character “1” (one)
+* A Bech32 version identifier
+  * The character “p” (version = 1) for on-chain with subaddress
+* A Bech32 encoded payload
+  * For version 1 is Libra account address + subaddress (16 + 8 bytes)
+The last 6 characters correspond to the Bech32 checksum
```

Comment:
You need another bullet here or to insert another newline because it combines this with the prior line right now